### DISCUSSION TITLE
Expected Release Date for v0.7

### DISCUSSION THREAD

**Discussion by lherrerai** [calliope-project/calliope#618] [General]
Hello Calliope Team,

First of all, thank you for creating such an intuitive and powerful tool. I have been using Calliope and it is incredibly helpful for my projects.

I was wondering if you could provide an estimated timeline for the release of v0.7?

Thanks again for your hard work and dedication to this project.

Best,  
Luis H

**Comment by brynpickering** [comment]
@lherrerai we're in the beta phase of testing - you can see lots of activity in the issues and PRs right now!

It takes time because we lack contributors _and_ testers - if you're happy to join in on either then we'd be happy to accept contributions :)

To test it, just install the latest development version according to [the documentation](https://calliope.readthedocs.io/en/v0.7.0.dev3/installation/) and try building the models you desire. The features are pretty much all there - we're just cleaning up around the edges and fixing some API bugs that lead to people getting unexpected results when they try creating/overwriting/updating the models.

If you have problems building your model or want to request we do things differently, then just raise an issue.

As to your original question: I wouldn't want to be drawn in on a specific timeline right now as we want to settle major API issues first. Until we get more testers on it, we can't know what to fix, so it's an impossible timeline to set.
